src/main/java/ecdar/controllers/EcdarController.java

```diff
@@ -732,6 +747,7 @@ private void initializeRecentProjectsMenu() {
                 try {
                     Ecdar.projectDirectory.set(path);
                     Ecdar.initializeProjectFolder();
+                    Ecdar.isSplitProperty().set(false);
```

Comment:
This is repeated in ```initializeOpenProjectMenuItem(...)``` maybe we should consider creating a function for this such as "setProjectDirectory(Path path);"?

Reply:
Very nice catch. I will do that